tools/actions_local_runner.py

```diff
@@ -60,7 +60,8 @@ def find_changed_files(merge_base: str = "origin/master") -> List[str]:
     cached = git(["diff", "--cached", "--name-only"])
 
     # Committed
-    diff_with_origin = git(["diff", "--name-only", "--merge-base", merge_base, "HEAD"])
+    merge_base = git(["merge-base", "origin/master", "HEAD"])[0]
```

Comment:
Given this line, what is the purpose of the `merge_base` parameter to this function now?

Reply:
None really, I took it out in the latest push